app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragment.java

```diff
@@ -311,7 +311,8 @@ public boolean setupClassMarkerClickListener(GoogleMap map) {
         map.setOnMarkerClickListener(new GoogleMap.OnMarkerClickListener() {
             @Override
             public boolean onMarkerClick(Marker marker) {
-                Log.i(TAG,marker.getTag().toString());
+                System.out.println(marker.getPosition());
```

Comment:
Don't we prefer logs over system out prints?

Reply:
yeah I forgot to remove it :s, I'll change it now